src/ravenpy/config/rvs.py

```diff
@@ -512,11 +514,11 @@ def zip(
         return zip
 
 
-def is_symbolic(params: dict) -> bool:
+def is_symbolic(params: Union[dict, Any]) -> bool:
     """Return True if parameters include a symbolic variable."""
     from pymbolic.primitives import Variable
 
-    if is_dataclass(params):
+    if is_dataclass(params) and not isinstance(params, type):
```

Comment:
Why is this change needed ?

Reply:
It's a membership check to ensure that we pass an instance of the class and not the `dataclass` class type. The justification is that the following operation iterating over `params` will work for a `dataclass` instance, but not the `dataclass` type.